__tests__/app.test.js

```diff
@@ -55,4 +54,41 @@ describe("General errors", ()=>{
       expect(body).toEqual({})
     })
   })
+})
+
+describe("GET /api/articles/:article_id", ()=>{
+  test("200: responds with an article object with correct properties", ()=>{
+    return request(app)
+    .get('/api/articles/1')
+    .expect(200)
+    .then(({body:{article}})=>{
+      console.log(article)
+        expect(article).toEqual({
```

Comment:
This works ok, but what happens if you add more keys to your response objects in the future? _Realistically_, are you going to go back and change every single test that begins to fail? Is there a more appropriate way of writing tests so that our code is scalable and open to future expansion?